src/Integration/LanguageGroup.cs

```diff
@@ -1,14 +1,15 @@
 //-----------------------------------------------------------------------
-// <copyright file="RuleSetGroup.cs" company="SonarSource SA and Microsoft Corporation">
+// <copyright file="LanguageGroup.cs" company="SonarSource SA and Microsoft Corporation">
 //   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
 //   Licensed under the MIT License. See License.txt in the project root for license information.
 // </copyright>
 //-----------------------------------------------------------------------
 
 namespace SonarLint.VisualStudio.Integration
 {
-    public enum RuleSetGroup
+    public enum LanguageGroup
```

Comment:
Why is this enum needed? It's bijective to the `Language` class members `CSharp`, `VBNET` and `Unknown` (which are singletons and also implement `IEquatable<Language>`).

The logical grouping of files (SonarQube rule set, additional files [not there yet]) is actually by _QualityProfile_.
At the moment SonarQube only supports _one QualityProfile per language_, for any given SQ project. So using `Language` as the grouping is reasonable*.

*unless SQ provides a way to specify different QPs for components within a SQ project, which they don't. At then at that time the grouping becomes the QP key/name/identifer/itself.